puppet/modules/web/manifests/vhost/rpm.pp

```diff
@@ -0,0 +1,106 @@
+# @summary Set up the rpm vhost
+# @api private
+class web::vhost::rpm (
+  String[1] $stable_foreman,
+  Stdlib::Fqdn $servername = 'rpm.theforeman.org',
+  Stdlib::Absolutepath $rpm_directory = '/var/www/vhosts/rpm/htdocs',
+  Stdlib::Absolutepath $rpm_source_directory = '/var/www/vhosts/stagingrpm/htdocs/',
+  String $user = 'rpmrepo',
+) {
+  $rpm_directory_config = [
+    {
+      path            => $rpm_directory,
+      options         => ['+Indexes', '+FollowSymLinks', '+MultiViews'],
```

Comment:
We don't need MultiViews. Perhaps if you disable this, you can disable more modules?
```suggestion
      options         => ['+Indexes', '+FollowSymLinks'],
```